fedimint-bitcoind/Cargo.toml

```diff
@@ -13,6 +13,7 @@ rustc-args = ["--cfg", "tokio_unstable"]
 
 [features]
 default = ["bitcoincore-rpc", "esplora-client"]
+fedimint-server = ["dep:fedimint-server-core"]
```

Comment:
Can you explain the reasoning behind this? Is it just so clients dont need to depend on `fedimint-server`? (I know we've talked about splitting this crate in two for awhile, one for client and one for server, this seems like a first step towards that)

Reply:
The code in `fedimint-bitcoind::shared` is really only server specific, and it interfers  with wasm client builds. I'll add a comment to explain it.